my-app/src/app/main-layout/content-projection/course/course.component.html

```diff
@@ -14,9 +14,9 @@
     <div class="col-lg">{{ course.description }}</div>
     <div class="col-lg text-center">
       <button class="btn btn-primary"
-        (click)="edit()">Edit</button>
+        (click)="edit()" id="edit-button">Edit</button>
```

Comment:
I consider giving ids for testing only is not a good idea. But it's my own opinion.
I prefer just calling method edit()
I think logic of event handling is a part of angular and it should be tested by angular.
Another opinion - unit tests as a defense from bugs. In this case, testing by clicking html element is a good point. So, it all depends on project and team I suppose.